fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -41,20 +41,97 @@
 typedef Reference<IDataDistributionTeam> ITeamRef;
 typedef std::pair<ITeamRef, ITeamRef> SrcDestTeamPair;
 
-// TODO: add guard to guarantee the priority is not equal for each purpose?
+// FIXME: Always use DataMovementReason to invoke these functions.
 inline bool isDiskRebalancePriority(int priority) {
 	return priority == SERVER_KNOBS->PRIORITY_REBALANCE_UNDERUTILIZED_TEAM ||
 	       priority == SERVER_KNOBS->PRIORITY_REBALANCE_OVERUTILIZED_TEAM;
 }
+
+inline bool isDiskRebalancePriority(DataMovementReason reason) {
+	return reason == DataMovementReason::REBALANCE_UNDERUTILIZED_TEAM ||
+	       reason == DataMovementReason::REBALANCE_OVERUTILIZED_TEAM;
+}
+
 inline bool isMountainChopperPriority(int priority) {
 	return priority == SERVER_KNOBS->PRIORITY_REBALANCE_OVERUTILIZED_TEAM ||
 	       priority == SERVER_KNOBS->PRIORITY_REBALANCE_READ_OVERUTIL_TEAM;
 }
+
+inline bool isMountainChopperPriority(DataMovementReason reason) {
```

Comment:
```suggestion
inline bool isDataMovementByMountainChopper(DataMovementReason reason) {
```

Reply:
fixed, but naming to isDataMovementForMountainChopper